### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by hershjac** [iopred/keystone#1]
I'm not entirely sure how you work, but would you either let me add a feature to display the alias of each keystone, or is that something you'd like to add?

**Comment by iopred** [comment]
Yeah dude, happy to accept a PR. I'll likely have a few comments, but that sounds like a great idea.

@Keystone dungeons or @Keystone aliases seems pretty reasonable to me.

**Comment by hershjac** [comment]
Sorry, I hadn't checked here in a few days, been a bit busy with random school work. I can maybe get something put up later this week. And yeah, I was going to go with @keystone aliases.
